### PR TITLE
Create kanishkasrivastava

### PR DESCRIPTION
**Pull request by kanishkasrivastava**
Fixes #

## Proposed Changes

-
-
-

### CODE REVIEW

**Review by aravindvnair99 — COMMENTED** [review]
@kanishkasrivastava The file is missing an extension.

**Comment by kanishkasrivastava** [comment]
What kind of extension please accept the pull requests

**Comment by kanishkasrivastava** [comment]
All changes done

**Review by kanishkasrivastava — COMMENTED** [review]
Happy hacktoberfest

File: kanishkasrivastava [hunk 1]
```diff
@@ -0,0 +1,125 @@
+Hello there
```
Comment (aravindvnair99) [inline kanishkasrivastava]:
```suggestion
```

Adding this here will lead to a compilation error.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@kanishkasrivastava Your file is still missing an extension. C++ files should end with `.cpp` extension.

**Review by kanishkasrivastava — COMMENTED** [review]
All issues solved

File: kanishkasrivastava.cpp [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+		if(type==4)
+		{
+			cout<<temp.CalculateVolume()<<endl;
+		}
+		if(type==5)
+		{
+			Box NewBox(temp);
+			cout<<NewBox<<endl;
+		}
+
```
Comment (aravindvnair99) [inline kanishkasrivastava.cpp:117]:
```suggestion
```

File: kanishkasrivastava.cpp [hunk 3]
```diff
@@ -0,0 +1,125 @@
+Hello there
```
Comment (aravindvnair99) [inline kanishkasrivastava.cpp:1]:
```suggestion
```

File: kanishkasrivastava.cpp [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+		}
+
+	}
+}
+
+int main()
+{
+	check2();
+}
+is a genral code for beginners in c++
```
Comment (aravindvnair99) [inline kanishkasrivastava.cpp:125]:
```suggestion
```

File: kanishkasrivastava.cpp [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+// int getBreadth (); // Return box's breadth
+// int getHeight ();  //Return box's height
+// long long CalculateVolume(); // Return the volume of the box
+
+//Overload operator < as specified
+//bool operator<(Box& b)
+
+//Overload operator << as specified
+//ostream& operator<<(ostream& out, Box& B)
+
```
Comment (aravindvnair99) [inline kanishkasrivastava.cpp:71]:
```suggestion
```

File: kanishkasrivastava.cpp [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+			Box NewBox(temp);
+			cout<<NewBox<<endl;
+		}
+
+	}
+}
+
+int main()
+{
+	check2();
```
Comment (aravindvnair99) [inline kanishkasrivastava.cpp:123]: Since this is just one function, code could be directly put here instead of calling a function from here.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

**Comment by aravindvnair99** [comment]
> All issues solved

@kanishkasrivastava Please take a look at the lines I have highlighted.

**Comment by aravindvnair99** [comment]
Duplicate of #29

**Comment by kanishkasrivastava** [comment]
DONE

**Pull request closed without merging**